src/models/supports.ts

```diff
@@ -58,7 +58,7 @@ export const SupportsModel = types
 
       getAllForSection(sectionId: string) {
         return self.supports.filter((support) => {
-          return (support.type !== SupportItemType.section) || (support.sectionId === sectionId);
+          return (support.type === SupportItemType.section) && (support.sectionId === sectionId);
```

Comment:
We don't have any way of showing unit/investigation/problem supports any more, but @lbondaryk will be adding the investigation reflection supports to a separate problem section